pkg/networkservice/common/refresh/client.go

```diff
@@ -108,16 +109,23 @@ func (t *refreshClient) Close(ctx context.Context, conn *networkservice.Connecti
 func after(ctx context.Context, conn *networkservice.Connection) (time.Duration, error) {
 	clockTime := clock.FromContext(ctx)
 
-	expireTime, err := ptypes.Timestamp(conn.GetCurrentPathSegment().GetExpires())
-	if err != nil {
-		return 0, errors.WithStack(err)
-	}
+	var minTimeout = time.Duration(math.MaxInt64)
```

Comment:
```suggestion
	var minTimeout time.Duration
```

Reply:
If we don't initialize `minTimeout` with max possible duration, it will always be `0`.

Reply:
Please fix the case when there no segments.

Reply:
Done